graphql_compiler/compiler/common.py

```diff
@@ -1,7 +1,7 @@
 # Copyright 2017 Kensho Technologies, LLC.
 from collections import namedtuple
 
-from . import emit_gremlin, emit_match, ir_lowering_gremlin, ir_lowering_match
```

Comment:
I would keep the names the same. We aren't lowering Gremlin or MATCH -- we are lowering IR for the sake of converting into Gremlin or MATCH.